src/isar/mission_planner/echo_planner.py

```diff
@@ -89,6 +88,21 @@ def _mission_plan(self, mission_id: int) -> dict:
 
         return response.json()
 
+    def _create_inspection_tasks_from_sensor_types(
+        self, tag_name: str, sensors: List[str]
+    ) -> List[Union[TakeImage, TakeThermalImage]]:
+        measurement_tasks: List[Union[TakeImage, TakeThermalImage]] = []
```

Comment:
I think it makes sense to just use `InspectionTask` i type hints here
```suggestion
    ) -> List[InspectionTask]:
        measurement_tasks: List[InspectionTask] = []
```

Reply:
Using InspectionTask instead of Union[...] gives a type hint error since mission.tasks is a List[TASKS].

Reply:
Ok, I get the same when I try here. Let's just keep it like this for now then.